.github/workflows/publish.yaml

```diff
@@ -38,9 +41,10 @@ jobs:
 
       - name: Import and push to Docker Hub
         run: |
+          version="$(cat rockcraft.yaml | yq e '.version')"
           sudo skopeo \
             --insecure-policy \
             copy \
-            oci-archive:charmed-mongodb_${{ github.ref_name }}_amd64.rock \
-            docker-daemon:dataplatformoci/charmed-mongodb:${{ github.ref_name }}
-          docker push dataplatformoci/charmed-mongodb:${{ github.ref_name }}
+            oci-archive:mongodb_${version}_amd64.rock \
+            docker-daemon:dataplatformoci/mongodb:${{ github.ref_name }}
```

Comment:
forgive me I've forgotten - but what is needed to change from `dataplatformoci` to the correct docker group

Reply:
The last order from Jon was to use GitHub docker repository and forget about DockerHub.

IMHO:
1) rename the repository to `charmed-mongodb-rock` (update [IaaS](https://github.com/canonical/data-platform))
2) switch CI/CD to publish to `ghcr.io/canonical/charmed-mongodb-rock:5-edge`,  see tech details [here](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages).